dcos-labs/ui-kit
# DCOS-40668: Fix create component template in UI kit

Fix component template paths

Closes DCOS-40668

## How to test

1. `npm run create:component blabla`
2. Go to `packages/index.ts` and export your module as so `export { Blabla } from "./blabla";`
3. `npm start` then visit `http://localhost:6006/`
4. Make sure you can see the component `blabla` on the left menu
